pyproject.toml

```
@@ -3,8 +3,8 @@ requires = [
     "setuptools >=61",
     "wheel >=0.38",
     "torch >=1.11",
-    "rascaline @ https://github.com/Luthaf/rascaline/archive/ff621db.zip",
     "cmake",
+    "rascaline@git+https://github.com/Luthaf/rascaline",
```

Please keep pinning to a specific commit here, just updating it to the latest one
```suggestion
    "rascaline @ https://github.com/Luthaf/rascaline/archive/881de9b.zip",
```